src/core/config.c

```diff
@@ -0,0 +1,139 @@
+#include "config.h"
+#include "xdpw.h"
+
+#include <dictionary.h>
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+#include <unistd.h>
+#include <wordexp.h>
+#include "iniparser.h"
+
+#define SYSCONFDIR "/etc"
+
+static const char *loglevels[] = {
```

Comment:
This is duplicated elsewhere. Maybe we can just use the one from `logger.c`, e.g. by introducing a helper function?

Reply:
good idea. wasn't myself happy with that.